crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -251,6 +251,10 @@ impl Relayer {
         // Then report update the internal state or report if submission failed
         match submission_result {
             Ok(height) => self.state_tx.send_modify(|state| {
+                debug!(
+                    da_height=%height,
```

Comment:
Not a fan of terse acronyms like this. I think just `%height` is descriptive enough given the context. Or, if necessary, `data_availability_height` or better `celestia_height` (since its the concrete height of the celestia network, not of some abstract DA network).